recipes/embree3/all/conanfile.py

```diff
@@ -60,8 +66,11 @@ class EmbreeConan(ConanFile):
         "ray_masking": False,
         "backface_culling": False,
         "ignore_invalid_rays": False,
+        "with_tbb": False,
     }
 
+    generators = "CMakeDeps"
+
```

Comment:
```suggestion
```

Please, avoiding mixing generators here, it makes harder to review. Move it to generate()